### ISSUE TITLE
Duplicate pips from Tile.getPIPs()?

### ISSUE THREAD

**Issue by litghost** [Xilinx/RapidWright#127]
Replication instructions (using #126):
```
git clone -b duplicate_pips git@github.com:litghost/RapidWright.git
cd RapidWright
make update_jars
RAPIDWRIGHT_PATH=$(pwd) ./scripts/invoke_rapidwright.sh com.xilinx.rapidwright.interchange.DeviceResourcesExample xc7a35tcpg236-1
```

Results in:
```
==============================================================================
==                  Device Resources Dump: xc7a35tcpg236-1                  ==
==============================================================================
             Load Device:     1.037s     60.164MBs
           populateEnums:     0.198s      4.692MBs                                                                                           
               SiteTypes:     0.163s      0.963MBs
Exception in thread "main" java.lang.RuntimeException: Duplicate pip!!!! 99 != 90, INT_L_X6Y10/INT_L.LVB_L0<<->>LVB_L12 == INT_L_X6Y10/INT_L.LVB_L0<<->>LVB_L12, 161 == 161, 160 == 160, 302404341 == 302404341, equals true
```

**Comment by clavin-xlnx** [comment]
Thank you, I have successfully reproduced the issue.  It appears that bi-directional PIPs are appearing twice in the results of calls to Tile.getPIPs().  I will work on providing a fix.

**Comment by clavin-xlnx** [comment]
An updated jar has been made with a fix for the duplicate PIP issue:
https://github.com/Xilinx/RapidWright/releases/download/v2020.2.1-beta/rapidwright-api-lib-2020.2.1_update1.jar

This will be added to the next release (2020.2.2).

**Comment by litghost** [comment]
@clavin-xlnx Has the latest JAR's been updated so that this issue is fixed?  If so, this issue can be closed!

**Comment by clavin-xlnx** [comment]
This fix is included in 2020.2.2.
